### DISCUSSION TITLE
Build and publish .deb files

### DISCUSSION THREAD

**Discussion by emmatyping** [brettcannon/python-launcher#146] [Ideas]
I recently set up an apt repo for various rust command line tools at https://apt.0xe.me, and I was going to add the python-launcher to it, I saw in https://github.com/brettcannon/python-launcher/issues/22 you were considering building deb/rpm/pkg packages. Are you still interested in building `.deb`s?

**Comment by brettcannon** [comment]
I'm interested if you do the work. 😉 The reason I closed that issue without `.deb` support is I use Linuxbrew and so my use-case was met and I was not going to be in a position to maintain it. But if you're up for doing the work I can try to make sure any critical changes like more shell completions end up in the configuration (or I ask you for help 😉).

**Reply by emmatyping** [reply]
I'm happy to work on making this happen, I'm currently planning on a github action to make this an easy addition to workflows. I think #148 will really only serve Fedora users. A lot of people not only run Debian, but use it as a base for docker containers and such.

**Reply by brettcannon** [reply]
I already have a workflow for building at release time, so all you should need to do is get the deb build working and then plug into the release workflow.
